src/routes/api/request.routes.js

```diff
@@ -12,5 +12,6 @@ const router = express.Router();
 
 router.post('/', [isUserLoggedInAndVerified, validateTripRequest], createTripRequest);
 router.get('/most-traveled-destinations', isUserLoggedInAndVerified, placesAndVisitTimes);
+router.get('/current-user-requests', isUserLoggedInAndVerified, getListOfMyRequests);
```

Comment:
@de-scholar could you use the `/trips` endpoint since you are working on a travel feature. For instance `GET /trips` will make more sense here

Reply:
@karamuka  the endpoint is `/api/trips/current-user-requests`